framework/src/transfers/MultiAppCopyTransfer.C

```diff
@@ -29,19 +30,19 @@ template<>
 InputParameters validParams<MultiAppCopyTransfer>()
 {
   InputParameters params = validParams<MultiAppTransfer>();
-  params.addRequiredParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.");
-  params.addRequiredParam<VariableName>("source_variable", "The variable to transfer from.");
-
+  params.addDeprecatedParam<AuxVariableName>("variable", "The auxiliary variable to store the transferred values in.", "This is being replaced by 'to_variable'.");
+  params.addDeprecatedParam<VariableName>("source_variable", "The variable to transfer from.", "This is replaced by 'from_variable'.");
+  params.addRequiredParam<VariableName>("to_variable", "The variable that the solution is being transfered into.");
+  params.addRequiredParam<VariableName>("from_variable", "The variable to transfer from.");
+  params.addClassDescription("Copies variables (nonlinear and auxiliary) between multiapps that have identical meshes.");
```

Comment:
Well... it's ok with me... but it's certainly outside the original intent.

The only thing I can think of that might be a little weird is the Transfer time interpolation stuff.  But I can't really see you Transferring to a nonlinear variable over time anyway (does that even make sense?  I guess maybe... as an initial guess for each timestep?)

Anyway - I'm not against this... I might even have some of my own uses for it....